[Bug]: Webpack bundle5 error even on fresh create-next-app@latest on node:lts image

### Describe the bug

Suddenly Storybook stopped working with the following error:

```console
ERROR in main
Module not found: TypeError: Cannot read properties of undefined (reading 'tap')

ERROR in Module not found: TypeError: Cannot read properties of undefined (reading 'tap')

ERROR in main
Module not found: TypeError: Cannot read properties of undefined (reading 'tap')

ERROR in   Error: Child compilation failed:
  Module not found: TypeError: Cannot read properties of undefined (reading 'tap  ')
  ModuleNotFoundError: Module not found: TypeError: Cannot read properties of un  defined (reading 'tap')
      at /my-app/node_modules/next/dist/compiled/webpack/bundle5.js:29:89507
      at /my-app/node_modules/next/dist/compiled/webpack/bundle5.js:29:426239
      at eval (eval at create (/my-app/node_modules/next/dist/compiled/webpack/b  undle5.js:14:9224), <anonymous>:10:1)
      at /my-app/node_modules/next/dist/compiled/webpack/bundle5.js:29:419889
      at eval (eval at create (/my-app/node_modules/next/dist/compiled/webpack/b  undle5.js:14:9224), <anonymous>:9:1)
      at /my-app/node_modules/next/dist/compiled/webpack/bundle5.js:29:424076
      at /my-app/node_modules/next/dist/compiled/webpack/bundle5.js:29:417323
      at /my-app/node_modules/next/dist/compiled/webpack/bundle5.js:29:424295
      at NormalModuleFactory.resolveRequestArray (/my-app/node_modules/next/dist  /compiled/webpack/bundle5.js:29:429278)
      at /my-app/node_modules/next/dist/compiled/webpack/bundle5.js:29:424237
[...] see file for more detailed output
```

I reproduced it on a fresh create-next-app@latest on bare node:lts image.
Settings where:
* nextjs (latest): typescript,  eslint, pages-router, no Tailwind, src dir,  turbopack  and no alias modification
* storybook (latest) minimal (only dev)


Added the output of `npm run storybook -- --debug-webpack`

[storybook.log](https://github.com/user-attachments/files/21886875/storybook.log)

### Reproduction link

reporduction link seems pointless (vanilla nextjs setub

### Reproduction steps

```console
root@4a76751659b0:/# docker -it --rm --name 'node-test' node:lts /bin/bash
[...]
root@4a76751659b0:/# npx create-next-app@latest
✔ What is your project named? … my-app
✔ Would you like to use TypeScript? …  Yes
✔ Which linter would you like to use? › ESLint
✔ Would you like to use Tailwind CSS? … No 
✔ Would you like your code inside a src/ directory? … Yes
✔ Would you like to use App Router? (recommended) … No
✔ Would you like to use Turbopack? (recommended) … Yes
✔ Would you like to customize the import alias (@/* by default)? … Yes
[...]
root@4a76751659b0:/# cd my-app
root@4a76751659b0:/my-app# npm create storybook@latest
✔ What configuration should we install? › Minimal: Component dev only
[...] 
# The error occurs 
root@4a76751659b0:/my-app# npm run storybook -- --debug-webpack >> storybook.log 2>&1 #  produced the above mentioned file
```


### System

```bash
Storybook Environment Info:

  System:
    OS: Linux 6.16 Debian GNU/Linux 12 (bookworm) 12 (bookworm)
    CPU: (8) x64 Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz
    Shell: 5.2.15 - /bin/bash
  Binaries:
    Node: 22.18.0 - /usr/local/bin/node
    Yarn: 1.22.22 - /usr/local/bin/yarn <----- active
    npm: 10.9.3 - /usr/local/bin/npm
  npmPackages:
    @storybook/nextjs: ^9.1.2 => 9.1.2 
    eslint-plugin-storybook: ^9.1.2 => 9.1.2 
    storybook: ^9.1.2 => 9.1.2
```

### Additional context

_No response_

--- a/code/frameworks/nextjs/src/utils.ts
+++ b/code/frameworks/nextjs/src/utils.ts
@@ -4,18 +4,18 @@ import { fileURLToPath } from 'node:url';
 
 import { getProjectRoot } from 'storybook/internal/common';
 
+import { WebpackDefinePlugin } from '@storybook/builder-webpack5';
+
 import type { NextConfig } from 'next';
 import { PHASE_DEVELOPMENT_SERVER } from 'next/constants.js';
 import nextJsLoadConfigModule from 'next/dist/server/config.js';
-import webpack from 'webpack';
 import type { Configuration as WebpackConfig } from 'webpack';
 
 import { resolvePackageDir } from '../../../core/src/shared/utils/module';
 
 export const configureRuntimeNextjsVersionResolution = (baseConfig: WebpackConfig): void => {
-  const { DefinePlugin } = webpack;
   baseConfig.plugins?.push(
-    new DefinePlugin({
+    new WebpackDefinePlugin({
       'process.env.__NEXT_VERSION': JSON.stringify(getNextjsVersion()),
     })
   );

